extensions/extensions-helm/aissemble-kafka-chart/pom.xml

```diff
@@ -15,6 +15,10 @@
 
     <build>
         <plugins>
+            <plugin>
```

Comment:
I: Now that we are using our own image again, we need to replace the image tag with the correct version at build time so releasing is smoother.